### DISCUSSION TITLE
posts/better-popcnt

### DISCUSSION THREAD

**Discussion by giscus** [EarthMessenger/blog.earthmessenger.xyz#4] [giscus]
# 2021/11/08/better-popcnt

众所周知 \(\mathrm{popcnt}\) 是一个十分常用的算法。

https://blog.earthmessenger.xyz/2021/11/08/better-popcnt.html

**Comment by zzyhzyy** [comment]
# %%%

**Comment by Enonya** [comment]
其实，严格意义上你不能把第一个的复杂度写成 $O(n)$ 吧）
写 $\log(n)$ 会好点*
